cookbook/form/form_customization.rst

```diff
@@ -282,7 +282,7 @@ can now re-use the form customization across many templates:
 
 .. code-block:: html+jinja
 
-    {# app/Resources/views/Form/fields.html.twig #}
+    {# src/AppBundle/Resources/views/Form/fields.html.twig #}
```

Comment:
this looks wrong to me. Best practices are to put templates in ``app/Resources/views``